README.md

```diff
@@ -52,6 +52,9 @@ In order to build this application, you need to install a few dependencies.
 
 #### Install Qt6
 
+Note: Currently the build requires 6.2.4. When you run the online installer, select
+6.2.4 version.
```

Comment:
Is it 6.2.4 or >= 6.2.4? We were just in a meeting where folks said we can build even with 6.4? cc @strseb

Reply:
Currently only 6.2.4 is considered compatible - I was only starting testing for 6.40 :D

Reply:
I couldn't get it to build on 6.4.0 on Mac.